<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>THAPlanningControls</class>
 <widget class="QWidget" name="THAPlanningControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>644</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Old model</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_currentConstruct">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Current construct</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_initialConstruct">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Initial construct</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_right">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>60</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_left">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>60</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>481</width>
         <height>66</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>偏距</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_hipOffset_currentMinusInit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>当前术侧 对比 对侧</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_hipOffset_opMinusContra">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>当前术侧 对比 术侧初始位</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>腿长</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_hipLength_opMinusContra">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_hipLength_currentMinusInit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>术侧初始位 对比 对侧</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEdit_hipLength_initMinusContra">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineEdit_hipOffset_initMinusContra">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>New model</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>161</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Femoral version test</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_right_new">
            <property name="text">
             <string>Right</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_left_new">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_femoralVersion">
          <property name="text">
           <string>Femoral version</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>166</width>
         <height>84</height>
        </rect>
       </property>
       <property name="title">
        <string>pelvisObject</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_initializePelvisObject">
          <property name="text">
           <string>Initialize PelvisObject</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_movePelvisObject">
          <property name="text">
           <string>set pelvisObject matrix</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>186</width>
         <height>113</height>
        </rect>
       </property>
       <property name="title">
        <string>femurObject</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_initializeLfemurObject">
          <property name="text">
           <string>Initialize L femurObject</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_initializeRfemurObject">
          <property name="text">
           <string>Initialize R femurObject</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_moveLfemurObject">
          <property name="text">
           <string>set L matrix</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_moveRfemurObject">
          <property name="text">
           <string>set R matrix</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>196</width>
         <height>142</height>
        </rect>
       </property>
       <property name="title">
        <string>reductionObject</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_noTiltCanalReduction">
          <property name="text">
           <string>noTilt canal reduction</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_noTiltMechanicReduction">
          <property name="text">
           <string>noTilt mechanix reduction</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_supineCanalReduction">
          <property name="text">
           <string>supineTilt canal reuction</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_supineMechanicReduction">
          <property name="text">
           <string>supineTilt mechanic reuction</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>0</y>
         <width>231</width>
         <height>113</height>
        </rect>
       </property>
       <property name="title">
        <string>stemObject and pelvisObject</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_initCupObject">
          <property name="text">
           <string>init cup</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_initStemObject">
          <property name="text">
           <string>init stem</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_moveCupObject">
          <property name="text">
           <string>move cup</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_moveStemObject">
          <property name="text">
           <string>move stem</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_changeCupObject">
          <property name="text">
           <string>change cup</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_changeStemObject">
          <property name="text">
           <string>change stem</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
